workflow/controller/controller.go

```diff
@@ -698,6 +691,12 @@ func (wfc *WorkflowController) newPodInformer() cache.SharedIndexInformer {
 				}
 			},
 			UpdateFunc: func(old, new interface{}) {
+				oldPod, newPod := old.(*apiv1.Pod), new.(*apiv1.Pod)
+				if oldPod.ResourceVersion == newPod.ResourceVersion {
+					wfc.metrics.PodResourceVersionRepeated()
```

Comment:
as above - I think think we'll ultimately want to capture the "repeated" metrics - I only included as I wanted t better understand the impact  I think you can delete that code